packages/editor/src/editor/FormToolbar.tsx

```diff
@@ -98,9 +99,9 @@ export const FormToolbar = forwardRef<HTMLDivElement>((_, ref) => {
           <PalettePopover label='Composite' icon={IvyIcons.File}>
             <CompositePalette />
           </PalettePopover>
-          <PalettePopover label='Data' icon={IvyIcons.DatabaseLink}>
-            <DataClassPalette />
-          </PalettePopover>
+          <PaletteElement text='Data'>
+            <DataClassDialog text='' iconVariant={undefined} className='no-border' worfkflowButtonsInit={false} />
+          </PaletteElement>
```

Comment:
the problem is that you use an `iconVariant` default. If you don't do that you don't have to set a special CSS class